src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,30 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string thread_name = capture_data_->GetThreadName(thread_state_slice->woker_tid());
+  std::string process_name = capture_data_->GetThreadName(thread_state_slice->woker_pid());
+  std::string description = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+  std::string woker_info;
+  if (thread_state_slice->woker_tid() != 0u) {
+    woker_info = absl::StrFormat(
+        "<br/>"
+        "<b>Woker process:</b> %s [%d]"
+        "<br/>"
+        "<b>Woker thread:</b> %s [%d]",
```

Comment:
I've just been talking to Tori about the term 'Woker' and it turns out this is not proper English. We think we should write `Was blocked by process:` and `Was blocked by thread:`.

Also an empty line after the thread state description would be nice.